### PR TITLE
Update network config, add GKE setup for db+server

### PR DESCRIPTION
**Pull request by pierewoj**

### CODE REVIEW

File: cloud/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  via `curl https://sdk.cloud.google.com | bash`)
+1. `gcloud` is initialized (if it's not then run `gcloud init` and proceed
+  with the instructions to log in and setup the project)
+1. Following APIs are enabled (if they are not, you can enable them
+  from the gcloud web ui): `container.googleapis.com`
+1. Ensure that you can push images to the GCP container registry
+  https://cloud.google.com/container-registry/docs/advanced-authentication
+
+# How to deploy changes
+1. `export PROJECT_NAME=YOUR_PROJ_NAME`
```
Comment (js361014) [inline cloud/README.md:13]: gcloud config set project <project>

File: cloud/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+This project contains the automation that sets up the service stack in GCP.
+
+# Requirements
+1. `gcloud` is installed (installation can be done
+  via `curl https://sdk.cloud.google.com | bash`)
+1. `gcloud` is initialized (if it's not then run `gcloud init` and proceed
+  with the instructions to log in and setup the project)
+1. Following APIs are enabled (if they are not, you can enable them
```
Comment (js361014) [inline cloud/README.md]: gcloud services enable container.googleapis.com

File: cloud/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  from the gcloud web ui): `container.googleapis.com`
+1. Ensure that you can push images to the GCP container registry
```
Comment (js361014) [inline cloud/README.md]: gcloud auth configure-docker

File: cloud/README.md (hunk 2 above)
Comment (pierewoj) [reply cloud/README.md]: done

File: cloud/README.md (hunk 3 above)
Comment (pierewoj) [reply cloud/README.md]: done

File: cloud/README.md (hunk 1 above)
Comment (pierewoj) [reply cloud/README.md:13]: done

**Pull request merged**